exporter/sematextexporter/README.md

```diff
@@ -0,0 +1,54 @@
+# Sematext Exporter
+<!-- status autogenerated section -->
+<!-- end autogenerated section -->
+
+This exporter supports sending metrics to [Sematext Cloud](https://sematext.com/) in Influx line protocol format
+
+## Configuration
+
+The following configuration options are supported:
+
+* `endpoint` (required) HTTP/S destination for metric receivers. It is dependent on the region:
+    - US: `spm-receiver.sematext.com`
+    - EU: `spm-receiver.eu.sematext.com`
+* `timeout` (default = 5s) Timeout for requests
+* `AppToken` App token is the token of Sematext Monitoring App to which you want to send the metrics.
+* `Region`  Region specifies the Sematext region the user is operating in; must be one of:
```

Comment:
Nitpick: In above line you say "you". Here "the user". Let's pick one and stick with it.